snippets/dart.json

```diff
@@ -100,6 +100,22 @@
 		"typedef": {
 			"prefix": "typedef",
 			"body": "typedef ${1:Type} ${2:Name}(${3:params});"
+		},
+		"test": {
+			"prefix": "test",
+			"body": [
+				"test('$1', () {",
+				"",
```

Comment:
Shouldn't there be a $2 inside here so you can jump to it easily? (eg. compare this to the `if` snippet)